mlflow/utils/validation.py

```diff
@@ -25,6 +25,9 @@
 # underscores, and dashes.
 _REGISTERED_MODEL_ALIAS_REGEX = re.compile(r"^[\w\-]*$")
 
+# Regex for valid registered model alias to prevent conflict with version aliases.
+_REGISTERED_MODEL_ALIAS_VERSION_REGEX = re.compile(r"^v\d+$")
```

Comment:
Is `v1.2.3` not reserved?

Reply:
What bout `V123`?

Reply:
We don't allow `'.'` in alias names which is caught by the previous regex, but good catch on the uppercase `'V'`.